main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -36,6 +36,12 @@ object Flix {
     * The reserved Flix delimiter.
     */
   val Delimiter: String = "%"
+
+
+  /**
+    * A case class to track the compile time spent in a phase and its sub-phases.
+    */
+  case class PhaseTime(phase: String, time: Long, subphases: List[(String, Long)])
```

Comment:
This is now public, so lets move it into the api directory proper. I don't like to refer to Flix.PhaseTime.